src/mocks/browser.js

```diff
@@ -8,4 +8,4 @@ const worker = setupWorker(
     }),
 )
 
-worker.start()
\ No newline at end of file
+// worker.start()
```

Comment:
what is the benefit of this? Purely out of curiosity.

Reply:
This is the mock service worker i was messing around with. It currently doesnt do anything. I should probably get in the habit of deleting test stuff like this.